internal/proxy/task.go

```diff
@@ -1337,6 +1338,23 @@ func (st *SearchTask) getVChannels() ([]vChan, error) {
 	return st.chMgr.getVChannels(collID)
 }
 
+// https://github.com/milvus-io/milvus/issues/6411
+// Support wildcard match
+func translateOutputFields(outputFields []string, schema *schemapb.CollectionSchema) ([]string, error) {
+	if len(outputFields) == 1 && strings.Trim(outputFields[0], " ") == "*" {
```

Comment:
better to use strings.TrimSpace, as "\t \n * \r\t" should also be "*" ? what do you think?

Reply:
Yeah, my fault here. I test strings.TrimSpace and strings.Trim(s, " "), two are equal, but strings.TrimSpace is more clear.